projects/igniteui-angular/schematics/ng-add/add-normalize.ts

```diff
@@ -0,0 +1,56 @@
+import * as path from 'path';
+
+import { Tree } from '@angular-devkit/schematics';
+import { getWorkspace, getWorkspacePath } from '@schematics/angular/utility/config';
+import { WorkspaceProject, ProjectType } from '@schematics/angular/utility/workspace-models';
+
+const cssPackage = { 'normalize.css': '^8.0.1' };
+const scssPackage = { 'normalize-scss': '^7.0.1' };
+
+export const cssImport = 'node_modules/normalize.css/normalize.css';
+export const scssImport = `@import "../node_modules/normalize-scss/sass/normalize";\n`
+    + `/* Standard CSS normalize, comment out if not required or using a different module */`
```

Comment:
Please add a newline to the end of the comment

Reply:
Also, for consistency, the import could be `"~/normalize-scss/sass/normalize"` instead of `"../node_modules/normalize-scss/sass/normalize"`

Reply:
I don't think .sass files support multi-line comments, so we should go with single line comments "//" instead of multi-line "/* */"